src/sqlite_back.cc

```diff
@@ -293,6 +293,7 @@ void SqliteBack::Bind(boost::spirit::hold_any v, DbTypes type, SqlStatement::Ptr
   CYCLUS_BINDVAL(MAP_STRING_STRING, std::map<std::string CYCLUS_COMMA std::string>);
   CYCLUS_BINDVAL(MAP_STRING_VECTOR_DOUBLE, std::map<std::string CYCLUS_COMMA std::vector<double> >);
   CYCLUS_BINDVAL(MAP_STRING_MAP_INT_DOUBLE, std::map<std::string CYCLUS_COMMA std::map<int CYCLUS_COMMA double> >);
+  CYCLUS_BINDVAL(MAP_STRING_PAIR_DOUBLE_MAP_INT_DOUBLE, std::map<std::string CYCLUS_COMMA std::pair<double CYCLUS_COMMA std::map<int CYCLUS_COMMA double> > >);
```

Comment:
Here is the diff:

```diff
   CYCLUS_BINDVAL(MAP_INT_STRING, std::map<int CYCLUS_COMMA std::string>);
   CYCLUS_BINDVAL(MAP_STRING_INT, std::map<std::string CYCLUS_COMMA int>);
   CYCLUS_BINDVAL(MAP_STRING_DOUBLE, std::map<std::string CYCLUS_COMMA double>);
-  CYCLUS_BINDVAL(MAP_STRING_STRING, std::map<std::string CYCLUS_COMMA std::string>);
-  CYCLUS_BINDVAL(MAP_STRING_VECTOR_DOUBLE, std::map<std::string CYCLUS_COMMA std::vector<double> >);
-  CYCLUS_BINDVAL(MAP_STRING_MAP_INT_DOUBLE, std::map<std::string CYCLUS_COMMA std::map<int CYCLUS_COMMA double> >);
-  CYCLUS_BINDVAL(MAP_STRING_PAIR_DOUBLE_MAP_INT_DOUBLE, std::map<std::string CYCLUS_COMMA std::pair<double CYCLUS_COMMA std::map<int CYCLUS_COMMA double> > >);
+  CYCLUS_BINDVAL(MAP_STRING_STRING,
+                 std::map<std::string CYCLUS_COMMA std::string>);
+  CYCLUS_BINDVAL(MAP_STRING_VECTOR_DOUBLE,
+                 std::map<std::string CYCLUS_COMMA std::vector<double> >);
+  CYCLUS_BINDVAL(
+      MAP_STRING_MAP_INT_DOUBLE,
+      std::map<std::string CYCLUS_COMMA std::map<int CYCLUS_COMMA double> >);
+  CYCLUS_BINDVAL(MAP_STRING_PAIR_DOUBLE_MAP_INT_DOUBLE,
+                 std::map<std::string CYCLUS_COMMA std::pair<
+                     double CYCLUS_COMMA std::map<int CYCLUS_COMMA double> > >)
```

I'm willing to make the change if you think the code is clearer/better with the line breaks.  I think it looks/functions better as is FWIW.